src/main/java/app/controller/ConfigController.java

```diff
@@ -69,6 +78,20 @@ public String getConfig(Model model) throws IOException {
     @PostMapping
     public String postConfig(@RequestParam final Map<String, Object> formParams) throws IOException {
         ConfigurationFileService.writeSettings(settingsPath, parseRootComponent.getRootComponent(formParams));
+              
+        // save data that is useful for other controllers to mongodb(db: application col: data)
+        Document applicationData= (new Document())
+        		.append("localNodePrivKey", formParams.get("privKeys"));
+    
+        final MongoCollection<Document> collection = mongoClient.getDatabase("application")
```

Comment:
use the database "apex" so that we stay on one db and dont have to wipe extra db definitions when we wipe the node